src/main/com/fulcrologic/rad/rendering/semantic_ui/components.cljc

```diff
@@ -42,10 +42,10 @@
 (defsc WrappedDropdown [this {:keys [onChange value multiple] :as props}]
   {:initLocalState (fn [this]
                      #?(:cljs
-                        (let [xform-options (memoize (fn [options]
```

Comment:
Why did you remove memoize? The transit conversion can be slow on large numbers of options. We should at least be using enc/memoize with a timeout, as this was added to address a real performance issue with real apps.